Source/WebKit/Shared/API/c/cf/WKStringCF.mm

```diff
@@ -54,12 +54,14 @@ CFStringRef WKStringCopyCFString(CFAllocatorRef allocatorRef, WKStringRef string
 {
     ASSERT(!WebKit::toImpl(stringRef)->string().isNull());
 
+    auto string = WebKit::toImpl(stringRef)->string();
+
     // NOTE: This does not use StringImpl::createCFString() since that function
     // expects to be called on the thread running WebCore.
     if (WebKit::toImpl(stringRef)->string().is8Bit()) {
```

Comment:
`if (string.is8Bit())`

Reply:
Oops. I missed this comment. I'll follow up now.

Reply:
https://github.com/WebKit/WebKit/pull/27920